src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -233,6 +239,14 @@ def load_command_table(self, _):
         g.custom_command('register', 'register_feature')
         g.custom_command('unregister', 'unregister_feature')
 
+    with self.command_group('feature registration', resource_feature_registration_sdk, client_factory=cf_feature_registrations, resource_type=PROFILE_TYPE,
+                            min_api='2021-07-01') as g:
+        feature_table_transform = '{Name:name, RegistrationState:properties.state}'
+        g.custom_command('list', 'list_feature_registrations', table_transformer='[].' + feature_table_transform)
+        g.show_command('show', 'get', table_transformer=feature_table_transform)
+        g.custom_command('create', 'create_feature_registration')
+        g.custom_command('delete ', 'delete_feature_registration')
```

Comment:
Please add confirmation for the deletion command `confirmation=True`

Reply:
removed

Reply:
after adding the confirmation=True, CI task started failing. As I have no idea how to fix it, I have to revert the change.

Reply:
What does confirmation=True do? Why did it break the CI tasks?

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#keyword-argument-reference

> confirmation - During interactive use, will prompt the user to confirm their choice to proceed. Supply a value of True to use the default prompt, or supply a string to use a custom prompt message. If the command is invoked in non-interactive scenarios and the --yes/-y parameter is not supplied, the command will fail.

The purpose of this parameter is to let the user confirm the deletion operation. It should not lead to the failure of CI. Could you show the specific error message when CI fails?

Reply:
thank you, seems it worked